js/apps/system/_admin/aardvark/APP/frontend/js/views/databaseView.js

```diff
@@ -176,34 +176,39 @@
       if (frontendConfig.forceOneShard) {
         sharding = 'single';
       }
-      var replicationFactor = $('#new-replication-factor').val();
-      var writeConcern = $('#new-write-concern').val();
-
-      var options = {
-        name: dbname,
-        "options" : {
-          "sharding" : sharding,
-          "replicationFactor" : Number(replicationFactor),
-          "writeConcern" : Number(writeConcern),
-        },
-        users: [{
-          username: userName
-        }]
-      };
-
-      this.collection.create(options, {
-        error: function (data, err) {
-          self.handleError(err, dbname);
-        },
-        success: function (data) {
-          if (window.location.hash === '#databases') {
-            self.updateDatabases();
+      
+      let replicationFactor = $('#new-replication-factor').val();
+      let writeConcern = $('#new-write-concern').val();
+
+      if(parseInt(writeConcern) <= parseInt(replicationFactor)) {
+        var options = {
+          name: dbname,
+          "options" : {
+            "sharding" : sharding,
+            "replicationFactor" : Number(replicationFactor),
+            "writeConcern" : Number(writeConcern),
```

Comment:
As we're parsing here to a numeric value again, I would create a small try-catch block before we reach the if statement where we can parse the value (and report and error in case it fails) and then execute the if block afterwards without the parse statement like:
```
if(writeConcern <= replicationFactor) {
```